### PR TITLE
Sets default number of bins to 5

### LINKED ISSUE #10357: Default number of bins for all BY VALUE numeric field options should be 5
If possible, please set the default number of bins to **5** for all BY VALUE numeric column options.

Every default for classification would be best at Quantiles, **5 bins**.

### PR DESCRIPTION
**Pull request by javierarce**
Closes #10357

This PR sets the default bins count of the numeric view of the Fill component to 5.

### CODE REVIEW

**Comment by javierarce** [comment]
CR: @xavijam

File: lib/assets/javascripts/cartodb3/components/form-components/editors/fill/input-number/input-number-value-content-view.js [hunk 1]
```diff
@@ -15,7 +15,10 @@ var MAIN_PAIN_INDEX = 1;
 var QUANTIFICATION_PANE_INDEX = 2;
 
 var QUANTIFICATION_METHODS = ['quantiles', 'jenks', 'equal', 'headtails'];
+var DEFAULT_QUANTIFICATION_METHOD_INDEX = 0;
+
 var BINS = ['2', '3', '4', '5', '6', '7'];
+var DEFAULT_BIN_INDEX = 3; // select '5' as the default bin size
```
Comment (javisantana) [inline lib/assets/javascripts/cartodb3/components/form-components/editors/fill/input-number/input-number-value-content-view.js]: I created a json file some time ago to have all of those things so anyone outside product team could change that easily https://github.com/CartoDB/cartodb/blob/master/lib/assets/javascripts/cartodb3/data/default-cartography.json

Comment (javierarce) [inline lib/assets/javascripts/cartodb3/components/form-components/editors/fill/input-number/input-number-value-content-view.js]: Good idea, I'll use a similar approach. Thanks!

**Comment by javierarce** [comment]
As suggested by @javisantana, I've added a `.json` file to handle the `bins` and `quantifications` settings for the fill component.

File: lib/assets/javascripts/cartodb3/components/form-components/editors/fill/default-fill-settings.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+  "number": {
+    "quantifications": {
+      "items": ['quantiles', 'jenks', 'equal', 'headtails'],
```
Comment (javierarce) [inline lib/assets/javascripts/cartodb3/components/form-components/editors/fill/default-fill-settings.json]: ups, thanks!

**Comment by javierarce** [comment]
Ok, fixed the quotes in the json file. If there's no other comment, once the build are green I'll merge it.

**Comment by xavijam** [comment]
:es:

**Pull request merged**